code/modules/mob/living/carbon/alien/alien.dm

```diff
@@ -207,6 +207,11 @@
 /mob/living/carbon/alien/setDNA()
 	return
 
+//This proc is NOT useless, we make it so that aliens have an halved siemens_coeff. Which means they take half damage
+/mob/living/carbon/alien/electrocute_act(const/shock_damage, const/obj/source, const/siemens_coeff = 0.5)

```

Comment:
This will only work if siemens_coeff itself is not set.
You should just multiply it by 0.5 in the proc itself, and then call the parent.